File: 프로그래머스/lv1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.java
class Solution {
    public long[] solution(int x, int n) {
        long[] answer = new long[n];
        
        // x씩 증가한다. -> x는 음수가 될 수도 있다.
        // x는 절대값으로 만들어서 체크해야된다.
        int plus = Math.abs(x);
        answer[0] = x;
        
        for(int i = 1 ; i < n ; i++){
            answer[i] = answer[i-1] + x;
        }
        
        
        
        return answer;
    }
}

File: 프로그래머스/lv1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.py
'''
    정수 x 와 자연수 n을 입력 받아 x  부터 x씩 증가하는 숫자 n개를 가지는 리스트 반환
'''
def solution(x, n):
    return [x*i for i in range(1, n+1)]
    
